src/lib/loaders/loaders_without_authentication/gravity.ts

```diff
@@ -76,6 +76,12 @@ export default (opts) => {
       { headers: true }
     ),
     collectionLoader: gravityLoader((id) => `collection/${id}`),
+    collectorProfilesLoader: gravityLoader(
```

Comment:
One of the few occurrences where we add this to the _without_ authentication to accomplish the task!

(usually it's b/c we're not personalizing or authorizing properly and respecting the current user, so the fix is typically to add the same-named loader to the _with_ authentication file! This of course makes sense b/c the use-case is a trusted app and not current user)

Reply:
yeah it felt....weird to add it here, thanks for the sanity check!